src/main/java/com/onarandombox/MultiverseCore/commands/DeleteCommand.java

```diff
@@ -0,0 +1,49 @@
+package com.onarandombox.MultiverseCore.commands;
+
+import co.aikar.commands.BukkitCommandIssuer;
+import co.aikar.commands.annotation.CommandAlias;
+import co.aikar.commands.annotation.CommandCompletion;
+import co.aikar.commands.annotation.CommandPermission;
+import co.aikar.commands.annotation.Conditions;
+import co.aikar.commands.annotation.Description;
+import co.aikar.commands.annotation.Single;
+import co.aikar.commands.annotation.Subcommand;
+import co.aikar.commands.annotation.Syntax;
+import com.onarandombox.MultiverseCore.MultiverseCore;
+import com.onarandombox.MultiverseCore.commandtools.queue.QueuedCommand;
+import org.bukkit.ChatColor;
+import org.jetbrains.annotations.NotNull;
+
+@CommandAlias("mv")
+public class DeleteCommand extends MultiverseCommand {
+    public DeleteCommand(@NotNull MultiverseCore plugin) {
+        super(plugin);
+    }
+
+    @Subcommand("delete")
+    @CommandPermission("multiverse.core.delete")
+    @Syntax("<world>")
+    @CommandCompletion("@mvworlds:type=includeunloaded")
+    @Description("Deletes a world on your server PERMANENTLY.")
+    public void onDeleteCommand(BukkitCommandIssuer issuer,
+
+                                @Single
+                                @Conditions("worldname:type=includeunloaded")
+                                @Syntax("<world>")
+                                @Description("Multiverse world you want to delete.")
```

Comment:
Don't think being a Multiverse world is required to be defined here

Reply:
Yeah take it or leave it I guess. Thought I do think this command only works on worlds managed by Multiverse?

Reply:
essentially if you are using an mv command its has to be an mv world (apart from import), it only works on mv worlds, so ig no need to specify **again** that it needs to be an mv world.